xml/ha_config_basics.xml

```diff
@@ -200,7 +243,82 @@
     </para>
    </important>
   </sect2>
+
+  <sect2 xml:id="sec.ha.config.basics.corosync.2-node">
+   <title>&corosync; Configuration for Two-Node Clusters</title>
+   <para>
+    When using the bootstrap scripts, the &corosync; configuration contains
+    a <literal>quorum</literal> section with the following options:
+   </para>
+   <example xml:id="ex.ha.config.basics.corosync.quorum">
+    <title>Excerpt of &corosync; Configuration for a Two-Node Cluster</title>
+    <screen>quorum {
+   # Enable and configure quorum subsystem (default: off)
+   # see also corosync.conf.5 and votequorum.5
+   provider: corosync_votequorum
+   expected_votes: 2
+   two_node: 1
+}</screen>
+   </example>
+   <para>
+    Normally you do not need to change this. Do not set
+    <literal>no-quorum-policy=ignore</literal> and <literal>two_node: 1</literal>
+    together. By default, when <literal>two_node: 1</literal> is set, the
+    <literal>wait_for_all</literal> option is automatically enabled.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec.ha.config.basics.corosync.n-node">
+   <title>&corosync; Configuration for N-Node Clusters</title>
+   <para>
+    As written before, we strongly recommend a two-node cluster or a N-node
```

Comment:
When not using a two-node cluster, we strongly recommend an odd number of nodes four your N-node cluster. With regards to quorum configuration, you have the following options: